### PR TITLE
updating lifestyle intervention documentation

### PR DESCRIPTION
**Pull request by catherineojohnson**

### CODE REVIEW

File: docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst [hunk 1]
```diff
@@ -1,23 +1,69 @@
...
+++++++++
+
+Participation in a structured program composed of education sessions and individualized coaching with the goal of reducing body weight and increasing physical activity for simulants. 
+
+
+Scenarios
++++++++++
+In the baseline scenario, no one is enrolled in structured lifestyle modification interventions.
+
+In the alternate scenarios, simulants are eligible for enrollment if their BMI was >= 25, they had no known DM2, and their FPG was 100-125mg/dl (or HbA1C 5.7-6.4). All simulants have a propensity score between 0 to 1 (uniform distribution). If they are eligible and their score is <0.5, the simulant is enrolled.
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst:14]:
```suggestion
In the alternate scenarios, 50% of eligible simulants are enrolled.  Simulants are eligible for enrollment if their BMI was >= 25, they had no known DM2, and their FPG was 100-125mg/dl (or HbA1C 5.7-6.4). All simulants have a propensity score between 0 to 1 (uniform distribution). If they are eligible and their score is <0.5, the simulant is enrolled. 
```

File: docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst [hunk 2]
```diff
@@ -1,23 +1,69 @@
...
+
+.. image:: ddpos_weight.jpg
 
 .. todo::
-  Address comment: see my comments on the vivarium_polypill.rst, I expect this doc to hit similar points relevant to lifestyle intervention.
+  Research team: Request data from the DDPOS investigators
 
-Decision tree and algorithm for lifestyle intervention
+.. todo::
+  Software engineers: The research team would prefer to use absolute change in weight, as it allows us to use the results from the meta-analysis described below to account for the decrease in SBP that occurs with weight reduction. This will require the software engineers to assign height values to all simulants as the inputs from GBD are BMI values.
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst:28]: Do we need to use a distribution of heights that research will provide?  Or will it be acceptable to assume everyone is precisely (for example) 2 meters tall.

**Review by aflaxman — APPROVED** [review]
This is a good start, but we will need to give the engineers more precise guidance on how to change the risk exposures over time for the simulants who receive this intervention.

**Review by ndecleene — APPROVED** [review]

File: docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst (hunk 2 above)
Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst:28]: Will provide distribution of heights/weights/BMI values from NHANEs

Comment (catherineojohnson) [reply docs/source/intervention_models/crm_mgmt/vivarium_lifestyle.rst:28]: format for data on height:
age sex year mean (height) std (height)

Could also look at the weighted ensemble distribution, but height should be mostly normally distributed, so okay to just assume that.

**Comment by aflaxman** [comment]
Collapsed by age, sex, year, pls

**Pull request merged**